packages/react/roving-focus/src/RovingFocusGroup.tsx

```diff
@@ -198,10 +203,20 @@ const RovingFocusGroupItem = React.forwardRef<RovingFocusItemElement, RovingFocu
   (props: ScopedProps<RovingFocusItemProps>, forwardedRef) => {
     const { __scopeRovingFocusGroup, focusable = true, active = false, ...itemProps } = props;
     const id = useId();
-    const context = useRovingFocusContext(ITEM_NAME, __scopeRovingFocusGroup);
+    const { onFocusableItemAdd, onFocusableItemRemove, ...context } = useRovingFocusContext(
+      ITEM_NAME,
+      __scopeRovingFocusGroup
+    );
     const isCurrentTabStop = context.currentTabStopId === id;
     const getItems = useCollection(__scopeRovingFocusGroup);
 
+    React.useEffect(() => {
+      if (focusable) {
+        onFocusableItemAdd();
+        return () => onFocusableItemRemove();
+      }
+    }, [focusable, onFocusableItemAdd, onFocusableItemRemove]);
```

Comment:
```suggestion
    const context = useRovingFocusContext(ITEM_NAME, __scopeRovingFocusGroup);
    const isCurrentTabStop = context.currentTabStopId === id;
    const getItems = useCollection(__scopeRovingFocusGroup);
    
    const { onFocusableItemAdd, onFocusableItemRemove } = context
    
    React.useEffect(() => {
      if (focusable) {
        onFocusableItemAdd();
        return () => onFocusableItemRemove();
      }
    }, [focusable, onFocusableItemAdd, onFocusableItemRemove]);
```

Nit: Small style thing but we prefer to keep `context` whole and destructure methods next to their effects when we need to please the linter

Reply:
Done.